packages/react-inbox/src/components/Messages/index.tsx

```diff
@@ -4,15 +4,17 @@ import Message from "../Message";
 import { InboxProps } from "../../types";
 import { Body, Footer, Header, Loading } from "./styled";
 
-import useMessages from "../../hooks/use-messages";
+import useInbox from "../../hooks/use-inbox";
+import useMessages from "~/hooks/use-messages";
 
 const Messages: React.FunctionComponent<InboxProps> = ({
   title = "Inbox",
   renderHeader,
   renderFooter,
   renderMessage,
 }) => {
-  const messageResults = useMessages();
+  useMessages();
```

Comment:
Does this need to be called on every render?

Reply:
Or if its not returning anything maybe it can located somewhere else

Reply:
its basically "useGetMessages" ~ i need to tweak this pattern and possibly rename